data/YuvrajChhetri.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Yuvraj Chhetri",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open source contributor | loves cloud computing | writes blogs | DevOps",
+  "avatar": "https://github.com/i-am-yuvi.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/uvource",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/yuvraj-chhetri-9ba17018a/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/i-am-yuvi",
```

Comment:
Hey @i-am-yuvi , you'll have to rename the file with your github username. i.e i-am-yuvi.json.

Reply:
Sure i'll do that